pandas/tests/test_sorting.py

```diff
@@ -181,13 +180,6 @@ def test_nargsort(self):
         exp = list(range(5)) + list(range(105, 110)) + list(range(104, 4, -1))
         tm.assert_numpy_array_equal(result, np.array(exp), check_dtype=False)
 
-    def test_nargsort_datetimearray_warning(self):
```

Comment:
should we not leave this?

Reply:
FWIW, I don't see the warning on master. It seems to have changed in the meantime. Probably fine to just remove.

Reply:
sure that's cool